### DISCUSSION TITLE
"Currently test data isn’t generated alongside a test generation"

### DISCUSSION THREAD

**Discussion by pledbetter-cvet** [forcedotcom/Einstein-GPT-for-Developers#74] [General]
Creating a new test class without a mechanism to create the required test data is not all that helpful. A little cart before the horse here...

**Comment by gbockus-sf** [comment]
Thanks for the feedback @pledbetter-cvet.  Converting to a discussion as that how we track feature requests/enhancements.  We'll continue to iterate and improve on our Test Case Generation feature.  In the mean time I'd recommend pairing it with inline autocomplete and side bar to 'fill the gaps'.

**Reply by patrykacc** [reply]
What are the capabilities of this model in terms of Unit Tests generation? I would appreciate any reference with some more detailed description, or bulleted list
